python/docs/docker.md

```diff
@@ -0,0 +1,34 @@
+# Docker
+
+## Build images
+
+We have currently two images to build:
+- taigaio/backend
+- taigaio/frontend
+
+Strong suggestion: tag them with `alpha` as they may suffer lots of changes and don't have backwards compatibility.
+
+To build them, it's recommended to use a fresh clone of the repository, only for this purpose. Then, from the root directory:
```

Comment:
Does it worth to introduce a link to the repo?

Reply:
it's this repo, but i can add a link